src/js/tracker.js

```diff
@@ -964,11 +964,6 @@
 				// Old Chrome versions add an unwanted requestEnd field
 				delete performanceTiming.requestEnd;
 
-				// Add the Chrome firstPaintTime to the performance if it exists
-				if (windowAlias.chrome && windowAlias.chrome.loadTimes && typeof windowAlias.chrome.loadTimes().firstPaintTime === 'number') {
-					performanceTiming.chromeFirstPaint = Math.round(windowAlias.chrome.loadTimes().firstPaintTime * 1000);
-				}
```

Comment:
isn't it safe to leave it for now while it's being deprecated. Maybe during the time it's deprecated the new standard will be implemented by other vendors?

Reply:
What's the impact of leaving it @miike? The check looks pretty defensive

Reply:
This is safe to leave in at the moment as the behaviour won't change until it's removed in M70 - so this PR can just sit here until before then.

Chrome has implemented a [PerformancePaintTiming API](https://developer.mozilla.org/en-US/docs/Web/API/PerformancePaintTiming) since M60 but it's marked experimental and the spec is still in progress. No word on other browsers implementing anything yet but with any luck that will happen before November.